app/code/Magento/Braintree/etc/adminhtml/menu.xml

```diff
@@ -7,13 +7,6 @@
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Backend:etc/menu.xsd">
     <menu>
-        <add
```

Comment:
Hi @Yogeshks. What the reason to move attributes to one line? As for me, the previous format is more readable. Also, Magento has the rules that check line length in the code.

Reply:
Hi @VladimirZaets, You're right but I had reviewed all `menu.xml` files inside `app/code/Magento` directory and found that `<add>` is written in one line inside all files. That is the only reason.

Reply:
@VladimirZaets, Please let me know if you still need updates.